.github/workflows/conformance.yml

```diff
@@ -53,11 +41,11 @@ jobs:
             envoy-image: "envoyproxy/envoy:v1.25-latest"
             consul-k8s-version: "v1.1.2"
             envoy-field: "apiGateway.imageEnvoy"
-          - name: "consul@v1.16-dev + consul-k8s@v1.2.0-rc1"
+          - name: "consul@v1.16 + consul-k8s@v1.2.0"
```

Comment:
Let's test with final release artifacts instead of the pre-release ones. In some cases, using the pre-release version was causing our CI to fail